### PR TITLE
3742 edit account

### PR DESCRIPTION
**Pull request by emitche**
Adds link to user menu to edit account

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h2>Edit <%= resource_name.to_s.humanize %></h2>
+<% content_for :title do 'Edit ' + resource_name.to_s.humanize end %>
```
Comment (hooverlunch) [inline app/views/devise/registrations/edit.html.erb]: Same thing with multiline blocks.

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -7,33 +7,43 @@
...
 
   <div class="actions">
-    <%= f.submit "Update" %>
+    <%= f.submit "Update", class: "btn btn-lg btn-primary" %>
   </div>
 <% end %>
+<br>
+<br>
 
 <h3>Cancel my account</h3>
```
Comment (hooverlunch) [inline app/views/devise/registrations/edit.html.erb]: Fuzzy says to please remove 'Cancel account'. Not needed.

File: app/views/devise/registrations/edit.html.erb [hunk 3]
```diff
@@ -7,33 +7,43 @@
...
     <%= f.password_field :current_password, autocomplete: "off" %>
   </div>
+  <br>
 
   <div class="actions">
-    <%= f.submit "Update" %>
+    <%= f.submit "Update", class: "btn btn-lg btn-primary" %>
   </div>
 <% end %>
+<br>
```
Comment (hooverlunch) [inline app/views/devise/registrations/edit.html.erb]: I talked to Fuzzy about form style. He says he is working on a standard form style for the app (on another feature branch). We agreed that we should merge this form as-is and once the standard style is done, go back and refactor this one. So when this PR is merged, can you please mark the issue as 'on hold' in redmine and add a note to this effect. Thanks!

**Comment by hooverlunch** [comment]
Rest looks good, thanks!

**Comment by emitche** [comment]
Need to do another last look over and then will merge into develop.

**Pull request merged**